schemas/Surgery Schema.json

```diff
@@ -84,7 +83,7 @@
                 "Craniotomy size (mm)"
             ]
         },
-        "injecties": {
+        "injects": {
```

Comment:
I guess if we're removing silly names, this should actually be `injections`.

Reply:
I don't want to overload names in these schemas, so I don't think injections is the right solution. I want to find a way to do this for defs consistently and just haven't come up with the right naming scheme. Maybe injection_defs. (or camel case if people prefer)